datasets/ONS_CPA/main.py

```diff
@@ -113,27 +108,21 @@ def product2cpa(p):
 cdids['Flow Directions'].cat.categories = cdids['Flow Directions'].cat.categories.map(
     lambda x: 'balance' if x == 'BAL' else 'imports' if x == 'IM' else 'exports' if x == 'EX' else None
 )
-# %%
-
-merged = pd.merge(df, cdids, on='cdid', how='left').drop_duplicates()
-merged.rename(columns={'cdid' : 'CDID'}, inplace=True)
-merged['Measure Type'] = "trade-in-goods"
-merged['Unit'] = "gbp-million"
-merged = merged[['ONS Partner Geography', 'Period', 'CDID', 'International Trade Basis',
-                             'Flow Directions', 'CPA 2008', 'Price Classification', 'Seasonal Adjustment','Value', 'Measure Type', 'Unit']].drop_duplicates()
+# -
+df = pd.merge(df, cdids, on='cdid', how='left').drop_duplicates()
+df.rename(columns={'cdid' : 'CDID'}, inplace=True)
+df['Measure Type'] = "trade-in-goods"
+df['Unit'] = "gbp-million"
```

Comment:
df['Measure Type'] = "trade-in-goods"
df['Unit'] = "gbp-million"
Could be deleted as it is a single cube and also Measure and Unit URIs are already coded in info.json.